File: .claude/skills/database-architecture/examples/javascript/mongoose/crud_example.ts
import mongoose from 'mongoose'

interface IUser extends mongoose.Document {
  name: string
  email: string
}

const UserSchema = new mongoose.Schema<IUser>({
  name: { type: String, required: true },
  email: { type: String, required: true, unique: true },
})

const User = mongoose.model<IUser>('User', UserSchema)

await mongoose.connect('mongodb://localhost:27017/demo')

const user = await User.create({ name: 'John', email: 'john@example.com' })
const users = await User.find()
console.log(users)

await User.findByIdAndDelete(user._id)

File: skills/database-architecture/examples/python/peewee/crud_example.py
from peewee import SqliteDatabase, CharField, Model

db = SqliteDatabase('demo.db')

class User(Model):
    name = CharField(max_length=100)
    email = CharField(max_length=255, unique=True)

    class Meta:
        database = db

db.create_tables([User])

user = User.create(name='John', email='john@example.com')
users = User.select()
User.get(User.name == 'John').delete_instance()